config/migrations/2023/20230613112721-add-politiezone-hulpverleningszone-flag.sparql

```diff
@@ -0,0 +1,84 @@
+    ?file schema:publication <http://lblod.data.gift/concepts/83f7b480-fcaf-4795-b603-7f3bce489325> .
+    ?submissionFile schema:publication <http://lblod.data.gift/concepts/83f7b480-fcaf-4795-b603-7f3bce489325> .
+    ?submissionDocumentFile schema:publication <http://lblod.data.gift/concepts/83f7b480-fcaf-4795-b603-7f3bce489325> .
+    ?formDataFile schema:publication <http://lblod.data.gift/concepts/83f7b480-fcaf-4795-b603-7f3bce489325> .
+  }
+  GRAPH ?j {
+    ?concept schema:publication <http://lblod.data.gift/concepts/83f7b480-fcaf-4795-b603-7f3bce489325> .
+  }
+}
+WHERE {
+  GRAPH ?g {
+    ?sub a meb:Submission ;
+        adms:status <http://lblod.data.gift/concepts/9bd8d86d-bb10-4456-a84e-91e9507c374c> ; # SENT
+        prov:generated ?form ;
+        pav:createdBy ?bestuurseenheid ;
+        dct:subject ?submissionDocument .
+
+
+    ?form <http://lblod.data.gift/vocabularies/besluit/authenticityType> |
+          <http://mu.semte.ch/vocabularies/ext/regulationType> |
+          <http://mu.semte.ch/vocabularies/ext/decisionType> |
+          <http://mu.semte.ch/vocabularies/ext/taxType> |
+          <http://data.europa.eu/eli/ontology#passed_by>/<http://data.vlaanderen.be/ns/mandaat#isTijdspecialisatieVan>/<http://data.vlaanderen.be/ns/besluit#bestuurt>/<http://data.vlaanderen.be/ns/besluit#classificatie> ?concept.
+
+    ?form
+        ext:decisionType <https://data.vlaanderen.be/id/concept/BesluitDocumentType/3fa67785-ffdc-4b30-8880-2b99d97b4dee> ; # Besluitenlijst
+        eli:passed_by ?bestuursorgaan .
+
+    OPTIONAL { ?submissionDocument adms:status ?submissionDocumentStatus . }
+    OPTIONAL { ?form <http://www.semanticdesktop.org/ontologies/2007/01/19/nie#hasPart> ?formDataFile . }
+    OPTIONAL { ?sub <http://www.semanticdesktop.org/ontologies/2007/01/19/nie#hasPart> ?submissionFile. }
+    OPTIONAL { ?submissionDocument <http://purl.org/dc/terms/source> ?submissionDocumentFile }
+
+  }
+
+  GRAPH <http://mu.semte.ch/graphs/public> {
+    VALUES (?bestuursorgaanClassificatieCode ?bestuurseenheidClassificatieCode) {
+        ( <http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/a9e30e31-0cd2-4f4a-9352-545c5d43be83> <http://data.vlaanderen.be/id/concept/BestuurseenheidClassificatieCode/ea446861-2c51-45fa-afd3-4e4a37b71562> )
+         ( <http://data.vlaanderen.be/id/concept/BestuursorgaanClassificatieCode/1afce932-53c1-46d8-8aab-90dcc331e67d> <http://data.vlaanderen.be/id/concept/BestuurseenheidClassificatieCode/a3922c6d-425b-474f-9a02-ffb71a436bfc> )
+    }
+
+    ?bestuursorgaan mandaat:isTijdspecialisatieVan/besluit:classificatie ?bestuursorgaanClassificatieCode .
+    ?bestuurseenheid besluit:classificatie ?bestuurseenheidClassificatieCode .
+    }
+
+  GRAPH ?i {
+    ?concept <http://www.w3.org/2004/02/skos/core#inScheme> |
+             <http://www.w3.org/2004/02/skos/core#topConceptOf> ?conceptScheme.
+
+    FILTER ( ?conceptScheme IN (
```

Comment:
I think here they are interested in all submissions from the bestuursorganen, so maybe no extra filter needed?